test/lsp/protocol_test_corpus.cc

```diff
@@ -694,4 +694,31 @@ TEST_CASE_FIXTURE(ProtocolTest, "ReportsSyntaxErrors") {
     CHECK_EQ(counters.getCategoryCounter("lsp.slow_path_reason", "changed_definition"), 0);
 }
 
+TEST_CASE_FIXTURE(ProtocolTest, "DidChangeConfigurationNotificationUpdatesHighlightUntypedSetting") {
```

Comment:
This test pretty much makes sure the feature runs without crashing for now. I would like to keep it to (a) have that sanity check, and (b) we just need to update assertDiagnostics so we can have a version that handles information diagnostics too,.

Reply:
Were you planning to be able to write a test that shows that the untyped diagnostics come back?